packages/server/api.js

```diff
@@ -1411,20 +1385,18 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
 
       if (Array.isArray(sessionIDs) === false) {
         response.status(400).end(JSON.stringify({
-          error: 'expected sessionIDs to be an array'
+          error: errors.INVALID_PARAMETER_TYPE
         }))
       } else if (sessionIDs.find(x => typeof x !== 'string')) {
         respones.status(400).end(JSON.stringify({
-          error: 'expected sessionIDs to be an array of strings'
+          error: errors.INVALID_PARAMETER_TYPE
```

Comment:
Clarification of the expected type here too would be nice. We could have a `error.invalidParameterType(key, expected, actual)` for generating these..?

Reply:
I totally want an `invalidParameterType` function, but I think it's a liiiittle out of the scope of this PR.

Anyways, I'll add custom messages to these two.